target/linux/generic/hack-6.6/645-netfilter-connmark-introduce-set-dscpmark.patch

```diff
@@ -209,6 +208,7 @@ Signed-off-by: Kevin Darbyshire-Bryant <ldir@darbyshire-bryant.me.uk>
 +		.targetsize     = sizeof(struct xt_connmark_tginfo3),
```

Comment:
Please split this struct into two parts to follow the upstream changes:
```
+	{
+		.name           = "CONNMARK",
+		.revision       = 3,
+		.family         = NFPROTO_IPV4,
+		.checkentry     = connmark_tg_check,
+		.target         = connmark_tg_v3,
+		.targetsize     = sizeof(struct xt_connmark_tginfo3),
+		.destroy        = connmark_tg_destroy,
+		.me             = THIS_MODULE,
+ 	},
+#if IS_ENABLED(CONFIG_IP6_NF_IPTABLES)
+	{
+		.name           = "CONNMARK",
+		.revision       = 3,
+		.family         = NFPROTO_IPV6,
+		.checkentry     = connmark_tg_check,
+		.target         = connmark_tg_v3,
+		.targetsize     = sizeof(struct xt_connmark_tginfo3),
+		.destroy        = connmark_tg_destroy,
+		.me             = THIS_MODULE,
+ 	},
+#endif
```
ref: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v6.6.57&id=997f67d813ce0cf5eb3cdb8f124da68141e91b6c

Reply:
Sorry, I do not understand what you're asking.

Reply:
@DragonBluep - Please see my latest commit and let me know if it is correctly split.

Reply:
yeah, I blew it.  Can you please provide me with a diff I can patch against my latest commit to fix it?